src/diff.ts

```diff
@@ -180,6 +188,22 @@ export function getDiff(
     }
   }
 
+  // violations
+  if (diff.chunks.bigger.length && bundleBudgets && bundleBudgets.length) {
```

Comment:
Trying to understand this but this portion of the code is essentially checking if we exceed our bundle %. How are we determining what's big or not? Are we using a fixed percentage that triggers the violation? I feel like it varies case by case

Reply:
say you have a bundle for the flag targeting page named `flagTargetingBundle` and its what 80 percent of the users navigate to, you need to protect the size of it because it can dictate if a User buys your product or not(evaluation period). We will assign a budget to only our critical bundles to protect ourselves from making them bigger than a certain threshold, a P.R, for example, shouldn't balloon the bundle by greater than 15 percent, if it does you have to figure out another way to implement your feature. this gives us an insurance policy against that.

Reply:
https://github.com/launchdarkly-labs/webpack-bundle-diff-action/blob/main/src/diff.ts#L17

this property tells us the delta percentage it changed (positive or negative)